packages/create-sitecore-jss/scripts/watch-templates.ts

```diff
@@ -40,6 +43,35 @@ function restoreLockfile() {
   }
 }
 
+/**
+ * Removes and then regenerates the yarn.lock file in order to resolve conflicting dependencies for react template.
+ */
+async function regenerateYarnLock() {
```

Comment:
Rename to match `restoreLockFile`?
```suggestion
async function regenerateLockFile() {
```